src/algorithms/classic/sample_based/rrt.py

```diff
@@ -33,8 +33,8 @@ def _get_new_vertex(self, q_near: Vertex, q_sample: Point, max_dist) -> Vertex:
 
     def _get_random_sample(self) -> Point:
         while True:
-            sample: Point = Point(torch.randint(0, self._get_grid().size.width, (1,)).item(),
-                                  torch.randint(0, self._get_grid().size.height, (1,)).item())
+            rand_pos = [torch.randint(0, self._get_grid().size[i], (1,)).item() for i in range(self._get_grid().size.n_dim)]  
+            sample: Point = Point(*tuple(rand_pos))
```

Comment:
No need for tuple here, it's already a list we can unpack
also, can we use np? We could have `rand_pos = np.random.randint(0, self.grid.size, self.grid.size.n_dim)`?